writlarge/templates/main/learningsite_detail.html

```diff
@@ -5,9 +5,12 @@
 {% block client_templates %}
     <meta name="csrf-token" content="{{csrf_token}}">
     {% include "main/client/minimal_map_template.html" %}
+    {% include "main/client/family_network.html" %}
 {% endblock %}
 
 {% block js %}
+    <script src='https://d3js.org/d3.v3.min.js'></script>
```

Comment:
Is this d3 addition intentional? I'm just wondering because I don't see any use of it elsewhere in this PR.

Reply:
good catch. I'm in the process of adding a d3 visualization. I'll remove for this PR.